symfony-csrf-protection-disabled.php

```
@@ -0,0 +1,148 @@
+// {/fact}
+
+// {fact rule=coral-csrf-rule@v1.0 defects=0}
+    // ok: symfony-csrf-protection-disabled
+    $resolver->setDefaults([
+      'data_class' => Type::class,
+    ]);
+// {/fact}
+
+// {fact rule=coral-csrf-rule@v1.0 defects=0}
+    // ok: symfony-csrf-protection-disabled
+    $resolver->setDefaults($options);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$options'. (undefined)

(UnusedLocalVariable)